### PR TITLE
Features/tcccd 83 jh

### PR DESCRIPTION
**Pull request by holmesjoli**
Very tiny changes

### CODE REVIEW

File: R/frontline_dm.R [hunk 1]
```diff
@@ -7,31 +7,15 @@ parse_date.frontline <- function(x, format = "\\d{4}-\\d{2}-\\d{2}"){
...
 #' @title Data management for reporting column
 #' @description Filter to only "reporting"
 #' @param df
 #' @return data.frame
 col.reporting <- function(df){
   
   df <- df %>% 
-    dplyr::mutate(reporting = ifelse(reporting_status == "Reporting", T, F)) %>% 
-    dplyr::filter(reporting)
+    dplyr::filter(reporting == "Reporting") %>%
```
Comment (sadierichardson) [inline R/frontline_dm.R]:
```suggestion
    dplyr::filter(reporting_status == "Reporting") %>% 
```

File: R/frontline_dm.R [hunk 2]
```diff
@@ -7,31 +7,15 @@ parse_date.frontline <- function(x, format = "\\d{4}-\\d{2}-\\d{2}"){
...
 #' @description Filter to only "reporting"
 #' @param df
 #' @return data.frame
 col.reporting <- function(df){
   
   df <- df %>% 
-    dplyr::mutate(reporting = ifelse(reporting_status == "Reporting", T, F)) %>% 
-    dplyr::filter(reporting)
+    dplyr::filter(reporting == "Reporting") %>% 
+    dplyr::select(-reporting)
```
Comment (sadierichardson) [inline R/frontline_dm.R]:
```suggestion
    dplyr::select(-reporting_status)
```

**Review by sadierichardson — CHANGES_REQUESTED** [review]
@holmesjoli just changed the variable name to match, but other than that, looks good!

**Comment by holmesjoli** [comment]
@sadierichardson good catch on both accounts. If you're ready to merge everything it all looks good to me!

**Pull request merged**